src/resource-archive/index.ts

```diff
@@ -46,26 +46,25 @@ class Watcher {
   private globalNetworkResolver: () => void;
 
   constructor(
-    private page: Page,
+    cdpClient: CDPSession,
```

Comment:
Is the `CDPSession` type from playwright?

Reply:
Yes, it is for now. I plan on common-izing the session type better when I run Cypress through this in a future PR.